src/pages/PlanPage/subRoutes/CurrentOrgPlan/InfoMessageCancellation/InfoMessageCancellation.tsx

```diff
@@ -14,19 +14,26 @@ function getPeriodEnd(
 function InfoMessageCancellation({
   subscriptionDetail,
 }: {
-  subscriptionDetail: z.infer<typeof SubscriptionDetailSchema>
+  subscriptionDetail?: z.infer<typeof SubscriptionDetailSchema>
 }) {
-  if (!subscriptionDetail?.cancelAtPeriodEnd) return null
-  const periodEnd = getPeriodEnd(subscriptionDetail)
+  let periodEnd: string | undefined
+  if (subscriptionDetail) {
+    if (!subscriptionDetail?.cancelAtPeriodEnd) return null
+    periodEnd = getPeriodEnd(subscriptionDetail)
+  }
   return (
-    <div className="col-start-1 col-end-13 mb-4">
-      <Message variant="info">
-        <h2 className="text-lg">Subscription Pending Cancellation</h2>
-        <p className="text-sm">
-          Your subscription has been cancelled and will become inactive on{' '}
-          {periodEnd}
-        </p>
-      </Message>
+    <div className="col-start-1 col-end-13 mb-4 sm:flex-initial md:w-2/3 lg:w-3/4">
+      <Alert variant="info">
+        <Alert.Title className="text-sm">Cancellation confirmation</Alert.Title>
+        <Alert.Description className="text-sm">
+          Your subscription has been successfully canceled. Your account{' '}
```

Comment:
cancelled vs. canceled -> only calling it out cuz the banner has 2 l's :P

Reply:
apparently both are correct and `canceled` is prefered in American English? I've always been more drawn to cancelled though until I googled this haha. I'm gonna change it back 🤷